internal/authentication/native.go

```diff
@@ -188,9 +188,9 @@ func UnmarshalECDHPrivateKey(privateScalar []byte) ECDHPrivateKey {
 	if sk.D.Cmp(elliptic.P256().Params().N) >= 0 {
 		return nil
 	}
-	x, y := sk.PublicKey.Curve.ScalarBaseMult(privateScalar)
```

Comment:
Are these changes required by the Go uprev? The `authentication` package is used internally and would ideally stay compatible with other Go versions.

Reply:
this changed was "forced" by the linter. alternatively we can turn off that rule. it was some

```
 QF1008: could remove embedded field "PublicKey" from selector (staticcheck)
        x, y := sk.PublicKey.Curve.ScalarBaseMult(privateScalar)
                   ^
```

let me know if you'd rather disable that rule